workshop/content/lab1.3_configure.md

```diff
@@ -141,3 +141,13 @@ tar -xf <file>
 ```
 kn version
 ```
+
+## `stern` CLI Installation
+
+`stern` is a great tool to easily be able to view logs for a particular container.
+
+1.  Download the newest release from [github.com](https://github.com/wercker/stern/releases).  Find your operating system and download it.
+
+2.  You should be able to double click the installer to install.
+
+Note: If you are on a macOS and have Homebrew installed, you can just run `brew install stern`.
```

Comment:
We should rework this from the perspective of running in homeroom.  They won't be installing `stern` on their own laptop.

Bonus: Extend the homeroom base image and install it via Dockerfile.  Then we don't need the user to install at all.

Reply:
@theckang 
Sounds great.  I'll remove the installation instructions and open another PR in the homeroom base image.  Do you happen to know where that repo lives?

Reply:
We chatted, turns out it's the `Dockerfile`, which I just finished updating.